bin/test

```diff
@@ -1,7 +1,24 @@
 #!/bin/bash
-set -euo pipefail
```

Comment:
I removed the `e` so that all the suites will always run

Reply:
And now of course only the last suite's exit status is used by Buildkite. I'll fix it on Monday. Tips appreciated... I'm not great at bash.

Reply:
I would use the approach here: https://stackoverflow.com/a/42219754

Example:

```
#!/bin/bash

err=0
trap 'err=1' ERR

echo "Good Command"
badcommand
echo "Good Command"

test $err = 0
```

This way if any one command errors out the script will return an error code on exit, but only after it reaches the last line.